pkg/lightning/backend/local/duplicate.go

```diff
@@ -1110,6 +1141,10 @@ func (local *DupeController) CollectRemoteDuplicateRows(
 		logger.End(zap.ErrorLevel, err)
 	}()
 
+	if !local.collectRemoteDupRows {
+		logger.Warn("[detect-dupe] skipping remote duplicate detection due to configuration")
+		return false, nil
```

Comment:
the outer logic depends on whether there are conflicts, please add a simple comment on the reason why it can be skipped